modules/lead/toolchain/rode-values.tpl

```diff
@@ -0,0 +1,30 @@
+grafeas:
+  certificates:
+    name: ${grafeas_cert}
+
+certificates:
+  name: ${rode_cert}
+
+rbac:
+  serviceAccountAnnotations:
+    eks.amazonaws.com/role-arn: ${iam_arn}
+
+localstack:
+  enabled: false
+
+resources:
+  limits:
+    cpu: 1
```

Comment:
this is pretty steep, what made you decide on using 1 cpu here?

Reply:
The `1 cpu` is actually the default limit in the Rode Helm chart, to be clear I was running into `OOM Killed` problems when testing in sandbox so I included the resources here to bump up the memory.

Reply:
We should probably change that in the rode chart then, but I think 1 CPU is definitely too hard.

Just so you understand, CPU usage doesn't have any bearing on whether or not a pod gets `OOM Killed`.  Since `OOM` stands for "out of memory".

Reply:
No problem with decreasing the cpu here, and I understood that the issue here was just with the memory; I just thought it was worth defining all the resource values we're using (_including the defaults from Rode_) for some more visibility.

I went ahead and did some more testing in sandbox to come up with more reserved values that performed well for me. I imagine these will adjust as we use proceed with dogfooding within our clusters.